tests/checkBoxes.spec.ts

```diff
@@ -1,80 +1,64 @@
-    await page.waitForSelector('.selected-specialties');
+    //3. Select the first veterinarian "Helen Leary" and click "Edit Vet" button
+    //Clicks seconds vet from the list
+    const edit_Vet_Button = page.getByText('Edit Vet')
+    edit_Vet_Button.nth(1).click()
 
+    //4. Add assertion of the "Specialties" field. The value "radiology" is displayed
     // Verify that 'radiology' is displayed in the selected specialties
-    const selectedSpecialties = await page.locator('.selected-specialties');
-    await expect(selectedSpecialties).toHaveText('radiology');
+    const specialties_Checkbox_Dropdown = page.locator('div.dropdown')
+    await expect(specialties_Checkbox_Dropdown).toContainText('radiology')
 
     //5. Click on the "Specialties" drop-down menu
-    await expandDropdownForCheckboxes;
+    await specialties_Checkbox_Dropdown.click()
 
     //6. Add assertion that "radiology" specialty is checked
-    expect(cardiologyCheckbox).toBeChecked()
+    // create const for values in checkbox
+    const radiology_Checkbox_Locator = page.locator('#radiology')
+    const dentistry_Checkbox_Locator = page.locator('#dentistry')
+    const surgery_Checkbox_Locator = page.locator('#surgery')
+    expect(radiology_Checkbox_Locator).toBeChecked()
     //7. Add assertion that "surgery" and "dentistry" specialties are unchecked
-    expect(dentistryCheckbox).not.toBeChecked()
-    expect(surgeryCheckbox).not.toBeChecked()
+    expect(dentistry_Checkbox_Locator).not.toBeChecked()
+    expect(dentistry_Checkbox_Locator).not.toBeChecked()
 
     //8. Check the "surgery" item specialty and uncheck the "cardiology" item speciality 
-    await expandDropdownForCheckboxes;
-    await waitForDropdownVisibility;
-    await surgeryCheckbox.check()
-    await cardiologyCheckbox.uncheck()
+    await surgery_Checkbox_Locator.check()
+    await radiology_Checkbox_Locator.uncheck()
 
     //9. Add assertion of the "Specialties" field displayed value "surgery"
-    await page.waitForSelector('.selected-specialties');
-    await expect(selectedSpecialties).toHaveText('surgery');
+    const specialties_Field = page.locator('.selected-specialties')
+    await expect(specialties_Field).toHaveText('surgery');
 
     //10. Check the "dentistry" item specialty
-    await expandDropdownForCheckboxes;
-    await waitForDropdownVisibility;
-    await dentistryCheckbox.check()
+    await dentistry_Checkbox_Locator.check()
 
     //11. Add assertion of the "Specialties" field. The value "surgery, dentistry" is displayed
-    await expect(selectedSpecialties).toHaveText('surgery, dentistry');
+    await expect(specialties_Field).toHaveText('surgery, dentistry');
 })
 
 test('Select all specialties', async ({ page }) => {
     //2. Select the third veterinarian "Rafael Ortega" and click "Edit Vet" button
-    await page.getByText('Edit Vet').nth(3).click()
+    const edit_Vet_Button = page.getByText('Edit Vet')
+    edit_Vet_Button.nth(3).click()
```

Comment:
Why did you create `edit_Vet_Button` constant?
It's not used anywhere later in the code so not needed. 
One line of code is better than two.